arrays.js

```diff
@@ -0,0 +1 @@
+console.log
```

Comment:
This should be formatted to print out a string in your console. So it should look something like this `console.log("heyo");`